stencil_main.py

```diff
@@ -180,31 +140,25 @@ def main(
         "test_gt4py",
     ]
     if stencil_name not in stencil_name_list:
-        print(
+        raise ValueError(
             "please make sure you choose one of the following stencil: {}".format(
                 stencil_name_list
             )
         )
-        sys.exit(0)
 
-    backend_list = ["numpy", "numbajit", "numbajit_inplace", "numbaloop", "numbastencil", "numbavectorize", "gt4py"]

```

Comment:
Not sure what `numbajit_inplace` means. `numbajit` is too generic since all Numba implementations leverage the JIT compiler. I suggest the following changes: 
- `numbajit` --> `numba_vector`
- `numbajit_inplace` --> removed
- `numbaloop` --> `numba_loop`
- `numbastencil` --> `numba_stencil`
- `numbavectorize` --> removed